sdk/spring/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/repository/SecondaryTestRepositoryConfig.java

```diff
@@ -43,12 +48,34 @@ public class SecondaryTestRepositoryConfig {
     @Value("${cosmos.secondary.responseContinuationTokenLimitInKb}")
     private int responseContinuationTokenLimitInKb;
 
+    @Value("${cosmos.diagnosticsThresholds.pointOperationLatencyThreshold}")
+    private int pointOperationLatencyThreshold;
+
+    @Value("${cosmos.diagnosticsThresholds.nonPointOperationLatencyThreshold}")
+    private int nonPointOperationLatencyThreshold;
+
+    @Value("${cosmos.diagnosticsThresholds.requestChargeThreshold}")
+    private int requestChargeThreshold;
+
+    @Value("${cosmos.diagnosticsThresholds.payloadSizeInBytesThreshold}")
+    private int payloadSizeInBytesThreshold;
+
     @Bean
     public CosmosClientBuilder secondaryCosmosClientBuilder() {
         return new CosmosClientBuilder()
             .key(cosmosDbKey)
             .endpoint(cosmosDbUri)
-            .contentResponseOnWriteEnabled(true);
+            .contentResponseOnWriteEnabled(true)
+            .clientTelemetryConfig(
```

Comment:
I thought in Spring, there is `ResponseDiagnosticsProcessor`. would there cause any overlap with the `diagnosticsHandler` ?

Reply:
I don't think there is any overlap, `ResponseDiagnosticsProcesor` is used to process the response from a query where as `diagnosticsHandler` is on the `CosmosClientBuilder`.

Any thoughts @kushagraThapar ?

Reply:
Actually, `ResponseDiagnosticsProcessor` is used for all operations in spring-data-cosmos. So it might overlap, we should let customers know to use `DiagnosticsHandler` on `CosmosClientBuilder` going forward. 
@trande4884  - we should add more samples / documentation on this in the readme section, and samples section.